src/components/fieldset/fieldset-test.stories.tsx

```diff
@@ -46,17 +54,9 @@ export const FieldsetStory = ({ legend, legendSpecialCharacters }) => {
   );
 };
 
-# Fieldset
-
-### Default
-
-<Canvas>
-  <Story
-    name="default"
-    args={{
-      legendSpecialCharacters: undefined,
-    }}
-  >
-    {FieldsetStory.bind({})}
-  </Story>
-</Canvas>
+DefaultStory.story = {
+  name: "default",
+  args: {
+    legendSpecialCharacters: undefined,
+  },
+};
```

Comment:
This pattern has now been deprecated in storybook versions 6+. From their [migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#hoisted-csf-annotations), Storybook are recommending we change it to this:
```suggestion
DefaultStory.storyName = "defaut";
DefaultStory.parameters = {
  args: {
    legendSpecialCharacters: undefined,
  },
};
```

Reply:
Good point, I suspect I don't need any of it as the story name is converted into the correct path but I'll include the updated pattern just in case a future update changes that